test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -419,20 +419,14 @@
                (testing "GET request"
```

Comment:
Might be good to change the testing string above a bit to indicate that we're expecting the url and url-and-method metrics to be absent because we intentionally disabled them when setting up the HTTP client.